User Interface/Tooltips/assets/xml/state_code.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<data>
+	
+	<!--blank-->
```

Comment:
If you don't specify an xml, it doesn't create the _ui object. Let me fix that with a quick edit...